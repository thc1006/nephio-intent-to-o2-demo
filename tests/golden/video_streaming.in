Set up high-bandwidth video streaming service for edge site 2 with CDN capabilities