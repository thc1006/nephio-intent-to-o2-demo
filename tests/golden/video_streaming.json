{
  "intentId": "intent_video_edge2",
  "name": "Video Streaming Service - Edge Site 2",
  "description": "Set up high-bandwidth video streaming service for edge site 2 with CDN capabilities",
  "targetSite": "edge2",
  "parameters": {
    "type": "video_streaming",
    "requirements": {
      "bandwidth": "10Gbps",
      "latency": "20ms",
      "capacity": "10000 concurrent users"
    },
    "configuration": {
      "service": "video_streaming",
      "cdn": true,
      "encoding": "h265",
      "quality": "4K"
    }
  },
  "priority": "high",
  "lifecycle": "draft"
}